configuration.yaml

```diff
@@ -29,6 +29,10 @@ CRISPR:
   inputDescriptionsTable: gs://otar000-evidence_input/CRISPR/data_files/crispr_descriptions.tsv
   inputCellTypesTable: gs://otar000-evidence_input/CRISPR/data_files/crispr_cell_lines_enriched_2021-10-22.tsv
   outputBucket: gs://otar000-evidence_input/CRISPR/json
+Essentiality:
+  inputBucket: gs://otar000-evidence_input/Essentiality/DepMap/2023Q4
+  depmap_tissue_mapping: https://raw.githubusercontent.com/opentargets/curation/master/mappings/biosystem/depmap_uberon_mapping.csv
```

Comment:
The `curation_repo` value can be interpolated here so that the file is fixed per release.